File: CodeJam/Python/reverseWords/reverseWords.py
#!/usr/bin/python
import os

def Main():
    output = 'output.txt'
    with open('large.in') as f:
        list = []
        next(f)
        for line in f:
            list.append( ' '.join(reversed(line.strip().split())) )
    try:
        os.remove(output)
    except OSError:
        pass

    with open(output, 'a') as f:
        for i in range(0, len(list)):
            f.write('Case #{0}: {1}\n'.format(i + 1, list[i]))


if __name__ == '__main__':
    Main()

File: CodeJam/JavaScript/reverseWords/reverseWords.js
const fs = require('fs');

const [, ...lines] = fs.readFileSync('large.in', 'utf8').trim().split('\n');
const output = lines.map(line => line.split(' ').reverse().join(' ')); 

const fd = fs.openSync('result.txt', 'a');

output.forEach((x, index) => {
	fs.writeSync(fd, `Case #${index + 1}: ${x}\n`);
});

fs.closeSync(fd);
